<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogHelpDBRead</class>
 <widget class="QDialog" name="DialogHelpDBRead">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>660</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::CustomContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Wymagany format DB odczyt</string>
  </property>
  <widget class="QPushButton" name="pushButtonCloseDialog">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>630</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Zamknij</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>604</width>
     <height>610</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="labelBit0">
        <property name="text">
         <string>bit 0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelBit1">
        <property name="text">
         <string>bit 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelBit2">
        <property name="text">
         <string>bit 2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelBit3">
        <property name="text">
         <string>bit 3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelBit4">
        <property name="text">
         <string>bit 4</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelBit5">
        <property name="text">
         <string>bit 5</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelBit6">
        <property name="text">
         <string>bit 6</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelBit7">
        <property name="text">
         <string>bit 7</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelFloat0">
        <property name="text">
         <string>float 0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelFloat1">
        <property name="text">
         <string>float 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelFloat2">
        <property name="text">
         <string>float 2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelFloat3">
        <property name="text">
         <string>float 3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelFloat4">
        <property name="text">
         <string>float 4</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelFloat5">
        <property name="text">
         <string>float 5</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelFloat6">
        <property name="text">
         <string>float 6</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelFloat7">
        <property name="text">
         <string>float 7</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelFloat8">
        <property name="text">
         <string>float 8</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelFloat9">
        <property name="text">
         <string>float 9</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_37">
        <property name="text">
         <string>0.0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_38">
        <property name="text">
         <string>0.1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_39">
        <property name="text">
         <string>0.2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_40">
        <property name="text">
         <string>0.3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_41">
        <property name="text">
         <string>0.4</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_42">
        <property name="text">
         <string>0.5</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_43">
        <property name="text">
         <string>0.6</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_44">
        <property name="text">
         <string>0.7</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_45">
        <property name="text">
         <string>2.0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_46">
        <property name="text">
         <string>6.0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_47">
        <property name="text">
         <string>10.0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_48">
        <property name="text">
         <string>14.0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_49">
        <property name="text">
         <string>18.0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_50">
        <property name="text">
         <string>22.0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_51">
        <property name="text">
         <string>26.0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_52">
        <property name="text">
         <string>30.0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_53">
        <property name="text">
         <string>34.0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_54">
        <property name="text">
         <string>38.0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>2</number>
      </property>
      <item>
       <widget class="QLabel" name="label_55">
        <property name="text">
         <string>Live bit - stan komunikacji</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_56">
        <property name="text">
         <string>Żądanie przesłania wymiarów elementu</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_57">
        <property name="text">
         <string>---</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_58">
        <property name="text">
         <string>---</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_59">
        <property name="text">
         <string>---</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_60">
        <property name="text">
         <string>---</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_61">
        <property name="text">
         <string>---</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_62">
        <property name="text">
         <string>---</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_63">
        <property name="text">
         <string>Grubość płyty</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_64">
        <property name="text">
         <string>---</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_65">
        <property name="text">
         <string>---</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_66">
        <property name="text">
         <string>---</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_67">
        <property name="text">
         <string>---</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_68">
        <property name="text">
         <string>---</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_69">
        <property name="text">
         <string>---</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_70">
        <property name="text">
         <string>---</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_71">
        <property name="text">
         <string>---</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_72">
        <property name="text">
         <string>---</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
